packages/config/src/ProfileInfo.ts

```diff
@@ -666,6 +788,49 @@ export class ProfileInfo {
         return argValue;
     }
 
+    // _______________________________________________________________________
+    /**
+     * Initialize a zosmf session configuration object with the arguments
+     * from profArgs
+     *
+     * @param profArgs
+     *      An array of profile argument attributes.
+     *
+     * @returns A session that can be used to connect to zosmf;
+     */
+    private static initZosmfSessCfg(profArgs: IProfArgAttrs[]): ISession {
+        const sessCfg: any = {};
+
+        // the set of names of arguments in IProfArgAttrs used in ISession
+        const profArgNames = [
+            "host", "port", "user", "password", "rejectUnauthorized",
+            "protocol","basePath", "tokenType", "tokenValue", "authtoken"
```

Comment:
`"authtoken"` -> `"authToken"`

Reply:
Good catch. This might have been very annoying to debug when a problem occurred.